sdk/communication/azure-communication-chat/azure/communication/chat/_shared/utils.py

```diff
@@ -103,5 +103,12 @@ def get_authentication_policy(
                     "or a token credential from azure.identity".format(type(credential)))
 
 def _convert_expires_on_datetime_to_utc_int(expires_on):
```

Comment:
Seems like you could have just called this existing function?

Reply:
The function name was sticking out since this method is being used to convert `datetime` to `int` and not anything specific to `expires_on` field. I have renamed it.